src/audio_unit/render_callback.rs

```diff
@@ -496,19 +496,15 @@ impl AudioUnit {
             let sample_rate: f64 = super::audio_session_get_property(id)?;
             (sample_rate * seconds as f64).round() as u32
         };
-        let mut data: Vec<u8> = vec![];
         let sample_bytes = stream_format.sample_format.size_in_bytes();
         let n_channels = stream_format.channels_per_frame;
         let data_byte_size = buffer_frame_size * sample_bytes as u32 * n_channels;
-        data.reserve_exact(data_byte_size as usize);
+        let mut data:Vec <u8> = vec![0u8; data_byte_size as usize];
```

Comment:
The type annotation is unneeded. I think we should have the right formatting as well.

Reply:
@ishitatsuyuki given you are here, do you want to rebase your original PR and take the credit?

Reply:
I just rebased #64.